packages/crayons-core/src/components/drag-container/readme.md

```diff
@@ -0,0 +1,230 @@
+# Drag Container (fw-drag-container)
+
+fw-drag-container can be used to reorder a list items via drag and drop or move/copy items between two fw-drag-container elements.
+
+## Demo
+
+fw-drag-container is a custom component with an open shadow that acts as a wrapper to the list items having `draggable` attribute as true, that can be dragged and dropped with in the wrapper to change their order.
+
+```html live
+<template>
+  <fw-drag-container>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Booking ID</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+    <div
+      draggable="false"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Customer Name</div>
+      <fw-icon class="fw-flex-grow-0" name="lock"></fw-icon>
+    </div>
+    <div
+      draggable="false"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Hotel</div>
+      <fw-icon class="fw-flex-grow-0" name="lock"></fw-icon>
+    </div>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Number of Rooms</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">ID Number</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Payment Type</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Room Preference</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+    <div
+      draggable="true"
+      class="fw-card-1 fw-p-24 fw-flex fw-flex-row fw-content-center"
+    >
+      <div class="fw-flex-grow fw-type-h5">Check</div>
+      <fw-icon class="fw-flex-grow-0" name="drag"></fw-icon>
+    </div>
+  </fw-drag-container>
+</template>
+```
```

Comment:
Can you add a sample to show how this can be used with our own options-list component?

Reply:
fw-drag-container is a wrapper around child items, so we can add any atomic list items like fw-icon, fw-tag, fw-select-option within this custom component. Since this is a custom component it doesn't work well with items inside slot.